lib/travis/build/addons/firefox.rb

```diff
@@ -68,13 +68,27 @@ def export_source_url
               'firefox-beta-latest'
             when 'latest-esr'
               'firefox-esr-latest'
+            when 'latest-dev'
+              'firefox-aurora-latest'
```

Comment:
What is `aurora`? Is there a chance that this is going to change? If so, how often does it change, and where should we look for the replacement?